dts/bindings/sensor/memsic,mc3419.yaml

```diff
@@ -40,3 +40,13 @@ properties:
       11 => LPF with fc set to IDR/12
       13 => LPF with fc set to IDR/16
       Default is disabling LPF.
+
+  decimation-rate:
+    type: int
+    default: 0
```

Comment:
Need to explain in the description why the default value was selected
https://docs.zephyrproject.org/latest/build/dts/bindings-upstream.html#dt-bindings-default-rules